back/node_watcher/src/tasks/types.ts

```diff
@@ -123,13 +124,28 @@ export enum ProposalStatus {
   TABLED = 'Tabled',
 }
 
-export enum ReferendumStatus {
-  CANCELLED = ' Cancelled',
-  EXECUTED = 'Executed',
-  NOTPASSED = 'NotPassed',
-  PASSED = 'Passed',
-  STARTED = 'Started',
-}
+export const referendumStatus = {
+  CANCELLED: 'Cancelled',
+  EXECUTED: 'Executed',
+  NOTPASSED: 'NotPassed',
+  PASSED: 'Passed',
+  STARTED: 'Started',
+};
+
+// export type ReferendumStatus =
+//   | 'Cancelled'
+//   | 'Executed'
+//   | 'NotPassed'
+//   | 'Passed'
+//   | 'Started';
```

Comment:
This is what I tried to keep using types, but I just like enum, I get annoyed by "type string can't be assigned to `ReferendumStatus`"